### DISCUSSION TITLE
Could anyone help me please in accessing the GitHub payment page.

### DISCUSSION THREAD

**Discussion by Balavenkatarao-paycraft** [community/community#144642] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### GitHub Feature Area

Other

### Body

Hello Team,

I am reaching out regarding an ongoing issue with processing a payment on GitHub. Each time I attempt to initiate the payment, I am redirected to a page displaying a unicorn logo (single-horn). This issue has persisted for the past 10 days, despite multiple attempts to complete the payment.

I raised this concern with GitHub Support 10 days ago and have been following up daily, yet I have not received any response. The situation has now resulted in surpassing the payment due date.

I would greatly appreciate any assistance or guidance to resolve this issue.

Regards
Bala

**Comment by jessie-ro** [comment]
Sorry to hear about your issue with the GitHub payment page. The unicorn logo typically points to a server-side issue, so while waiting for support to respond, you could try clearing your browser cache, switching to a different browser or device, or checking GitHub's [Status Page](https://www.githubstatus.com/) for outages. If the issue persists, reaching out to GitHub Support on Twitter (@GitHubSupport) might get a quicker response. Double-checking your payment method with your bank could also help. Hope this gets resolved soon!

**Comment by queenofcorgis** [comment]
Hi @Balavenkatarao-paycraft, 

This community is managed by GitHub employees, but it is not a support forum. The only place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact) or trying the methods @jessie-ro helpfully suggested.

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. 

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
